app/src/main/java/com/gholem/moneylab/main/MainActivity.kt

```diff
@@ -25,12 +25,24 @@ class MainActivity : AppCompatActivity() {
         setContentView(binding.root)
         setupBottomNavigation()
 
-        showBottomNavigation(savedInstanceState?.getBoolean(BOTTOM_NAVIGATION_VISIBILITY_KEY) ?: false)
+        showBottomNavigation(
+            savedInstanceState?.getBoolean(BOTTOM_NAVIGATION_VISIBILITY_KEY) ?: false
+        )
     }
 
+    //Can change stuff when screen is rotating(or when screen is reloaded)
     override fun onSaveInstanceState(outState: Bundle) {
         super.onSaveInstanceState(outState)
-        outState.putBoolean(BOTTOM_NAVIGATION_VISIBILITY_KEY, binding.bottomNavigationView.isVisible)
+        outState.putBoolean(
+            BOTTOM_NAVIGATION_VISIBILITY_KEY,
+            binding.bottomNavigationView.isVisible
+        )
+    }
+
+    //TODO BackBtn to return into navigation
+    override fun onBackPressed() {
+
```

Comment:
Please remove unnecessary blank lines in every while which is edited in this Pull Request - it can be irritating at the beginning but you will appreciate this when there will be more code in the class and this code will be more complicated. More info about the code style:

https://developer.android.com/kotlin/style-guide
https://kotlinlang.org/docs/coding-conventions.html